x-pack/plugin/core/src/main/java/org/elasticsearch/xpack/core/security/action/apikey/GetApiKeyRequest.java

```diff
@@ -36,29 +36,6 @@ public final class GetApiKeyRequest extends ActionRequest {
     private final boolean withLimitedBy;
     private final boolean activeOnly;
 
-    public GetApiKeyRequest(StreamInput in) throws IOException {
-        super(in);
-        realmName = textOrNull(in.readOptionalString());
-        userName = textOrNull(in.readOptionalString());
-        apiKeyId = textOrNull(in.readOptionalString());
-        apiKeyName = textOrNull(in.readOptionalString());
-        if (in.getTransportVersion().onOrAfter(TransportVersions.V_7_4_0)) {
-            ownedByAuthenticatedUser = in.readOptionalBoolean();
-        } else {
-            ownedByAuthenticatedUser = false;
-        }
-        if (in.getTransportVersion().onOrAfter(TransportVersions.V_8_5_0)) {
-            withLimitedBy = in.readBoolean();
-        } else {
-            withLimitedBy = false;
-        }
-        if (in.getTransportVersion().onOrAfter(API_KEY_ACTIVE_ONLY_PARAM_TRANSPORT_VERSION)) {
```

Comment:
nit: we should also remove `API_KEY_ACTIVE_ONLY_PARAM_TRANSPORT_VERSION` since it's not used anymore.